backend/core/forms.py

```diff
@@ -66,3 +67,22 @@ def get_email_addresses_from_str(self, email_addresses_str):
         email_addresses = re.split("[ ,\n]", email_addresses_str)
         email_addresses = [email.strip() for email in email_addresses]
         return [email for email in email_addresses if email]
+
+
+class AddGithubCollaboratorForm(forms.Form):
+    include_complete_projects = forms.BooleanField(
+        label="Would you like to include complete projects?",
+        required=False,
+        initial=False,
+    )
```

Comment:
![image](https://github.com/Umuzi-org/Tilde/assets/54069197/b53eda50-4525-4ca6-ba2a-e6b0f04dcdfc)

UX wise: I think this should not be a question and the checkmark should be before the text. so maybe:
```
[  ] include complete projects
```

I'm thinking more close to forms where you have to tick to accept terms and conditions or you want to be sent promotional emails.

I just don't think it should be a question since the only answer option is to say yes, maybe if it was a toggle or it had to radio inputs